api/openapi_server/__main__.py

```diff
@@ -24,7 +24,7 @@ def main():
                 arguments={'title': 'Home Unite Us'},
                 pythonic_params=True)             
     app.add_error_handler(AuthError, handle_auth_error)
-    app.run(port=8080, debug=True)
+    app.run(port=8080, debug=True, use_reloader=False)
```

Comment:
When this isn't enabled, I can't get the vscode debugger to start flask app. I suspect it's the debugger that's unable to attach to the subprocess created by Werkzeug. I included some background info in the resources but I think it's only to have functionality of making updates to the app while it's live. If we're on ci/cd, I'm assuming we'd have to re-deploy anyway? As for local development, I'm thinking that restarting the server with your new changes is a small price for being able to debug requests. Open to any thoughts here.